openstack/swift/templates/health-exporter-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if $.Values.health_exporter }}
+{{- if not $.Values.health_statsd }}
+{{- range $cluster_id, $cluster := .Values.clusters }}
```

Comment:
Once per region is enough. We don't need that per swift-cluster as there is only identical information.